### PR TITLE
New `is` operator option for blocklist Windows signatures

### PR DESCRIPTION
**Pull request by joepeeples**
Resolves https://github.com/elastic/security-docs/issues/5732. Updates both ESS 8.16 (currently `main`) and serverless docs.

## Previews
Step 4c/4iii in procedure on these pages:
- ESS: [Blocklist](https://security-docs_bk_5780.docs-preview.app.elstc.co/guide/en/security/master/blocklist.html)
- Serverless: use the [link below](https://github.com/elastic/security-docs/pull/5780#issuecomment-2329879339), then in the Security card click **View serverless docs**.  Navigate to **Manage Elastic Defend → Blocklist**.

### CODE REVIEW

**Comment by elasticdocs** [comment]
:rocket: Built _elastic-dot-co-docs-preview-docs_ successfully!

* https://elastic-dot-co-docs-production-c8tzwg4a9-elastic-dev.vercel.app

* Built with commit 57b3959227d052d9bfcff627cf025d85c5ce2ddd

> Issues? Visit #next-docs in Slack

**Comment by joepeeples** [comment]
> @joepeeples Shouldn't this PR include API changes as well?

Hi @szwarckonrad, it varies feature by feature whether Engineering or Docs updates the API docs. Often the engineer has a fuller understanding of what needs to be changed, but I see you provided details in the [docs issue](https://github.com/elastic/security-docs/issues/5732) so I can add an update to this PR, hopefully later today.

**Comment by joepeeples** [comment]
@szwarckonrad API docs updated in https://github.com/elastic/security-docs/pull/5780/commits/c3943a3d453a0870af01c163a7576c9c769dbc57. Ready for review!

**Review by szwarckonrad — DISMISSED** [review]
Thank you!

**Review by benironside — APPROVED** [review]

File: docs/management/api/blocklist-api.asciidoc [hunk 1]
```diff
@@ -525,8 +525,8 @@ Process signature is supported for Windows xref:exception-{endpoint-artifact-api
 * `entries`: An array with 1 `entry` item.
 * `entries[0]` : The entry defining the signature to be matched upon:
 ** `field`: Must be set to `subject_name`.
-** `value` : An array of signature names to match on.
-** `type` : Must be set to `match_any`.
+** `value` : To match multiple signatures, set to an array of signature names. To match a single signature, set to the signature name as a string.
```
Comment (natasha-moore-elastic) [inline docs/management/api/blocklist-api.asciidoc:528]:
```suggestion
** `value` : To match multiple signatures, specify an array of signature names. To match a single signature, specify the signature name as a string.
```

**Review by natasha-moore-elastic — DISMISSED** [review]
One optional nit, otherwise LGTM!

**Review by jmikell821 — COMMENTED** [review]
A +1 for Nat's suggestions; LGTM otherwise!

**Review by natasha-moore-elastic — APPROVED** [review]

**Pull request merged**